observer/test/observer.test.js

```diff
@@ -10,25 +11,23 @@ describe('observer', () => {
       pgPools = await getPgPools()
     })
 
-    it('observes scheduled rewards', async () => {
+    beforeEach(async () => {
       await pgPools.evaluate.query('DELETE FROM daily_participants')
       await pgPools.evaluate.query('DELETE FROM participants')
-      const { rows: insertRows } = await pgPools.evaluate.query(`
-        INSERT INTO participants
-        (participant_address)
-        VALUES
-          ('0xCURRENT'),
-          ('0xOLD')
-        RETURNING id
-      `)
-      await pgPools.evaluate.query(`
-        INSERT INTO daily_participants
-        (participant_id, day)
-        VALUES
-          ($1, now()),
-          ($2, now() - interval '4 days')
-      `, [insertRows[0].id, insertRows[1].id])
+      const today = new Date()
+      await givenDailyParticipants(
+        pgPools.evaluate,
+        `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`,
```

Comment:
I believe you can use a Date instance directly; the pg client or the pg server is smart enough to auto-convert the full date-time value to a date.

Reply:
should we adjust the type then, to accept string|Date?